cpp/src/arrow/compute/kernels/hash_aggregate.cc

```diff
@@ -2129,9 +2179,10 @@ Result<FieldVector> ResolveKernels(
 
 Result<std::unique_ptr<Grouper>> Grouper::Make(const std::vector<ValueDescr>& descrs,
                                                ExecContext* ctx) {
-  if (GrouperFastImpl::CanUse(descrs)) {
-    return GrouperFastImpl::Make(descrs, ctx);
-  }
+  // TODO(niranda) re-enable this!
+  //  if (GrouperFastImpl::CanUse(descrs)) {
+  //    return GrouperFastImpl::Make(descrs, ctx);
+  //  }
```

Comment:
Can you make a JIRA for this?

Reply:
Oh!!! I completely forgot about this TBH! :-( This needs to be added before merging this PR. I was waiting for #10858 PR to be merged to add this change!